app/src/main/java/com/example/pomo/activities/ItemActivity.kt

```diff
@@ -0,0 +1,50 @@
+package com.example.pomo.activities
+
+import android.os.Bundle
+import android.widget.TextView
+import android.widget.Toast
+import androidx.appcompat.app.AppCompatActivity
+import androidx.room.PrimaryKey
+import com.example.pomo.R
+import com.example.pomo.model.PomodoroSession
+import kotlinx.android.synthetic.main.activity_create_pomo_item.*
+import org.w3c.dom.Text
+import java.io.Serializable
+import java.util.*
+
+
+
+class ItemActivity:AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_item_information)
+        val item = intent.getSerializableExtra("POMOITEM") as PomodoroSession
+       // Toast.makeText(this, item.toString(), Toast.LENGTH_SHORT).show()
```

Comment:
```suggestion
```

Reply:
this toast was for testing purposes to display object to be passed to next activity